src/rebuild/srv.c

```diff
@@ -338,26 +450,30 @@ rebuild_global_status_update(struct rebuild_global_pool_tracker *rgt,
 	if (!is_rebuild_global_scan_done(rgt)) {
 		rebuild_leader_set_status(rgt, iv->riv_rank, iv->riv_dtx_resyc_version,
 					  SCAN_DONE);
-		D_DEBUG(DB_REBUILD, "rebuild ver %d tgt %d scan done\n",
-			rgt->rgt_rebuild_ver, iv->riv_rank);
+		D_DEBUG(DB_REBUILD, DF_RB ": rank %d scan done\n", DP_RB_RGT(rgt), iv->riv_rank);
 		/* If global scan is not done, then you can not trust
 		 * pull status. But if the rebuild on that target is
 		 * failed(riv_status != 0), then the target will report
 		 * both scan and pull status to the leader, i.e. they
 		 * both can be trusted.
 		 */
-		if (iv->riv_status == 0)
+		if (iv->riv_status == 0) {
+			/* test only: update_and_warn_for_slow_engines(rgt); */
```

Comment:
the comment can be removed?

Reply:
Yes, preferably I can do this cleanup along with a couple of other items in a future patch.